CHANGELOG.md

```diff
@@ -18,9 +18,11 @@ All notable changes to this project will be documented in this file.
 
 - Remove unused environment variable `AIRFLOW_UID` from the Airflow
   image ([#429]).
+- Java base image: hard-coded security properties for DNS caching ([#433])
```

Comment:
Do we want to have a dedicated CHANGELOG for the java base image?
```suggestion
- Java base image: Re-enable DNS caching ([#433])
```

Reply:
IMO no. The current situation shows that multiple changelogs don't work. I personally want a single changelog that is maintained - especially with the release entries.

Also I think my entry above is more accurate than your suggestion because the plan is to delegate these settings to the operators - which might decide to keep the cache disabled.

Reply:
Agreed. Wasn't aware of the operator setting the dns settings in the future.
WDYT of
`- Java base image: Remove hard-coded JVM security properties containing DNS caching settings. Going forward operators will configure DNS caching settings ([#433])`

Reply:
Done https://github.com/stackabletech/docker-images/pull/433/commits/017e0ed4d2bd56891d836c5e4da0e03e7eccede4